frontend/src/toolbar/experiments/ExperimentsEditingToolbarMenu.tsx

```diff
@@ -0,0 +1,208 @@
+import { IconPlus, IconSearch, IconTrash } from '@posthog/icons'
+import { LemonDivider } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { Form, Group } from 'kea-forms'
+import { EditableField } from 'lib/components/EditableField/EditableField'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+
+import { ToolbarMenu } from '~/toolbar/bar/ToolbarMenu'
+import { experimentsTabLogic } from '~/toolbar/experiments/experimentsTabLogic'
+import { WebExperimentTransformField } from '~/toolbar/experiments/WebExperimentTransformField'
+
+export const ExperimentsEditingToolbarMenu = (): JSX.Element => {
+    const { selectedExperimentId, inspectingElement, experimentForm } = useValues(experimentsTabLogic)
+    const {
+        setExperimentFormValue,
+        selectExperiment,
+        selectVariant,
+        inspectForElementWithIndex,
+        deleteExperiment,
+        newVariant,
+        newElement,
+        removeElement,
+        removeVariant,
+    } = useActions(experimentsTabLogic)
+
+    return (
+        <ToolbarMenu>
+            <Form
+                name="experiment"
+                logic={experimentsTabLogic}
+                formKey="experimentForm"
+                enableFormOnSubmit
+                className="flex flex-col overflow-hidden flex-1"
+            >
+                <ToolbarMenu.Header className="border-b">
+                    <h1 className="p-1 font-bold text-sm mb-0">
+                        {selectedExperimentId === 'new' ? 'New ' : 'Edit '}
+                        experiment
+                    </h1>
+                </ToolbarMenu.Header>
+                <ToolbarMenu.Body>
+                    <div>
+                        <div className="flex">
+                            {selectedExperimentId === 'new' ? (
+                                <EditableField
+                                    onSave={(newName) => {
+                                        experimentForm.name = newName
+                                    }}
+                                    name="item-name-small"
+                                    value={experimentForm.name}
+                                />
+                            ) : (
+                                <h4>{experimentForm.name}</h4>
+                            )}
+                            <LemonButton
+                                type="secondary"
+                                size="small"
+                                className="ml-3"
+                                sideIcon={<IconPlus />}
+                                onClick={newVariant}
+                            >
+                                Add Variant
```

Comment:
Sentence case please: https://posthog.com/handbook/growth/marketing/posthog-style-guide#use-sentence-case-for-titles

```suggestion
                                Add variant
```